packages/unit-testing/README.md

```diff
@@ -32,14 +34,243 @@ module.exports = SuiteCloudJestConfiguration.build({
 ```
 
 ## Installation
+> ⚠ SuiteCloud Unit Testing is installed as a `devDependency`.
+
 To install SuiteCloud Unit Testing in your project, from your project folder, run the following command:
 ```
 npm install @oracle/suitecloud-unit-testing --save-dev
 ```
 
 ➡ If you use SuiteCloud CLI for Node.js, you can install SuiteCloud Unit Testing when running the [`project:create`](../../packages/node-cli/README.md/##Commands) command by following the questions prompted.
 
-⚠ SuiteCloud Unit Testing is installed as a `devDependency`.
+## SuiteCloud Unit Testing Examples
+
+Here you can find two examples on how to use SuiteCloud Unit Testing with a SuiteCloud project.
+
+The first example covers testing for the **N/record** module, which is fully mocked in SuiteCloud Unit Testing. Whereas the second example covers the testing of a module that is not mocked in SuiteCloud Unit Testing, by using a custom stub.
+
+>💡 You can manually mock any module that is still not supported in SuiteCloud Unit Testing.
+
+### N/record Module Example
+
+Make sure your project follows a structure similar to the one presented below:
```

Comment:
I am not sure about this sentence, since this is an example and not a guide maybe it's better to say smth. like this instead:
Given the project structure similar to the one presented below